scripts/busybox-ping-test/helper_functions.py

```diff
@@ -60,18 +105,36 @@ def get_mac_id(HOST):
     mac_addr = addr_string[addr_string.index('ether') + 1]
     return mac_addr
 
+def get_username():
+   #return pwd.getpwuid( os.getuid() )[ 0 ]
+   return 'ubuntu'
+
```

Comment:
I forgot to ask about this change. What does the user ubuntu signify here, it is not the logged in user, "jenkins" user. Waht is this return value used for?

Reply:
You are right, the script uses this user 'ubuntu' to log on to the target machine to carry on its operations. The module pwd.getpwuid() return 'jenkins' when run from jenkins. So we would then need the jenkin's public key to be copied over to our targets to work. Since ubuntu is a default user that we use on all three hosts in questions, so it makes it easier to run the tests using that account.